cmake/config/config_gbs-tizen.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeForceCompiler)
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR arm)
```

Comment:
@haesik We use the form of "_{architecture}-{os}_.cmake". What about `config_arm-tizen-gbs`?

Reply:
I don't agree with your opinion.. Currently, considering gbs as an architecture is now easy to support gbs build.. if we uas config_arm-tizen-gbs, we should make config_arm-tizen-gbs, config_x86-tizen-gbs, config_x86_64-tizen-gbs, etc,,.. for the same gbs build.